GenderPayGap.WebUI/Views/Home/AccessibilityStatement.cshtml

```diff
@@ -0,0 +1,122 @@
+﻿@using GenderPayGap.Core
+@{
+    ViewBag.Title = "Accessibility Statement - Gender pay gap reporting service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row" xmlns="http://www.w3.org/1999/html">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Accessibility statement for Gender pay gap service
+        </h1>
+        <p class="govuk-body">
+            We want everyone to be able to do what they need to do with our website, regardless of their access needs, due to a disability or condition.
+        </p>
+        
+        <h2 class="govuk-heading-m">A detailed accessibility statement about the Gender pay gap service</h2>
+        <p class="govuk-body">
+            This page describes:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>how accessible our website is</li>
+            <li>what to do if you have a problem</li>
+            <li>what we are doing to meet the regulations</li>
+        </ul>
+        <p class="govuk-body">
+            This accessibility statement applies to the Gender pay gap service.
+        </p>
+        
+        <h2 class="govuk-heading-m">
+            Using our website and services
+        </h2>
+        <p class="govuk-body">
+            This service is run by the Government Equalities Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>change colours, contrast levels, zoom and fonts</li>
+            <li>complete tasks without a mouse by using input controls such as speech or a keyboard</li>
+            <li>listen using a screen reader</li>
+        </ul>
+        <p class="govuk-body">
+            We’ve also made the website text as simple as possible to understand.
+        </p>
+        <p class="govuk-body">
+            AbilityNet has advice on making your device easier to use if you have a disability.
+        </p>

```

Comment:
In the Google Doc, Tor commented that we might not want this.

Can you check with Tor and either:
* remove this paragraph
OR
* make `AbilityNet` a link to the AbilityNet website

Reply:
I looked at what she was referring to here (she sent me the original GOV.UK version of the accessibility statement) and I think she meant that she wasn't sure whether to make `AbilityNet` a link? On the GOV.UK version it isn't but this line is still in there - happy to clarify this with her! :)